libvast/native-plugins/rest_api/export.cpp

```diff
@@ -0,0 +1,237 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2022 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/format/json.hpp"
+
+#include <vast/command.hpp>
+#include <vast/concept/convertible/to.hpp>
+#include <vast/concept/parseable/numeric.hpp>
+#include <vast/concept/parseable/to.hpp>
+#include <vast/concept/parseable/vast/expression.hpp>
+#include <vast/db_version.hpp>
+#include <vast/plugin.hpp>
+#include <vast/query_context.hpp>
+#include <vast/system/actors.hpp>
+#include <vast/system/node_control.hpp>
+#include <vast/system/query_cursor.hpp>
+#include <vast/table_slice.hpp>
+
+#include <caf/stateful_actor.hpp>
+#include <caf/typed_event_based_actor.hpp>
+
+namespace vast::plugins::rest_api::export_ {
+
+// clang-format off
```

Comment:
If you comment the handlers similar to how we do it in actors.hpp you won't need this.

Reply:
I know, but I still don't like the idea of adding more work to fool a tool into behaving as if it was turned off, instead of just turning it off :D